formapp/templates/controlpanel.html

```diff
@@ -144,15 +167,18 @@ <h2>Users</h2>
     <button id="collapseButton" class="inline popOut hidden">Collapse all</button>
 
     <div id="students">
+
     </div>
 
     <h2>Restart session</h2>
     <button class="inline popOut" onclick="restart();">Remove all students and restart</button><br>
   </div>
   <div id="tabs-2">
     <h2>Number of students</h2>
-    <input type="number" id="studentNum" class="line" onchange="sendStudents();" name="studentNum" min="1" max="30">&nbsp;&nbsp;
-    <input type="checkbox" id="autoStudents" checked onchange="sendStudents();"><label for="autoStudents">Auto</label><br>
+    <input type="number" id="studentNum" class="line" onchange="sendStudents();" name="studentNum" min="1"
+      max="30">&nbsp;&nbsp;
+    <input type="checkbox" id="autoStudents" checked onchange="sendStudents();"><label
+      for="autoStudents">Auto</label><br>
```

Comment:
Clean up some of these multiline html tags and make them one line. The formatter might be doing this.